### ISSUE TITLE
Roadmap for v2 and v3

### ISSUE THREAD

**Issue by justinyoo** [Azure/azure-functions-openapi-extension#357]
> This is the list of features, enhancements, and bug fixes to be released into the upcoming releases, especially with breaking changes.
> 
> This is not the coplete list, which can be changing at any time.

## V2 ##

* Power Platform Custom Connector supports:
    * #36
    * #231

## V3 ##

* System.Text.Json support:
    * #154

**Comment by cloud-devlpr** [comment]
Is it possible to generate OpenAPI doc by tags, similar to the web interface, using CLI azfuncopenapi

**Comment by justinyoo** [comment]
> Is it possible to generate OpenAPI doc by tags, similar to the web interface, using CLI azfuncopenapi

Yes, it's possible. Find this [doc](https://github.com/Azure/azure-functions-openapi-extension/blob/main/docs/openapi.md#swagger-ui-endpoints-filtering).

**Comment by usamaa9** [comment]
Can we include support for generating open api doc from xml documentation #348 ? This would be really useful and personally I think it is better than using Attributes to describe models etc

**Comment by DiaAWAY** [comment]
> Can we include support for generating open api doc from xml documentation #348 ? This would be really useful and personally I think it is better than using Attributes to describe models etc

I agree, we're already very used to using XML comments to decorate our swagger/openapi documents from source code, and we do this extensively in our current functions and asp.net solutions, and it makes it easier when sharing logic and (particularly) models across solutions (primarily asp.net and azure functions + durable functions) which we then generate swagger files for APIM. 

Older swagger generation support enabled us to include XML documentation if we desired to do so, in addition XML documentation is also what is described for [ASP.NET openapi/swagger](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-6.0&tabs=visual-studio), of course azure functions isn't asp.net - but it would be nice if the Microsoft eco system agreed on how to annotate models for openapi/swagger generation instead of creating new annotations in different packages for different frameworks.

**Comment by justinyoo** [comment]
We don't currently have a concrete roadmap to support XML comments for now like what Swashbuckle offers. But we'll keep that discussion open.

**Comment by johnnyreilly** [comment]
It would be great to have a way to validate that the schema generated and the underlying function don't go out of sync. There's a discussion partway down this issue on the topic:

https://github.com/Azure/azure-functions-openapi-extension/issues/350#issuecomment-1034528702

Might actually be good to open this as a distinct issue, as each time I look for the discussion I struggle to find it!

**Comment by sajagjain** [comment]
@justinyoo It has been a year now. Do you have it in the roadmap now?

**Comment by Craigology** [comment]
@justinyoo Hope you're well 👋 What's the current position regarding official support for .NET 8? Thanks.

**Comment by justinyoo** [comment]
@Craigology We're supporting .NET 8 as well. The current version 1.5.1 works on .NET 8 RC1, though.

**Comment by KarenTazayan** [comment]
Hi @justinyoo, is this project retired? I haven't seen any new releases since January.

**Comment by Mivaweb** [comment]
Any news on this? Also the request for using the `FullNamespace` is required for some of us. Not even available in the 2.0.0- preview releases...

**Comment by mattchenderson** [comment]
We are not planning additional features for this project. Please see https://github.com/Azure/azure-functions-openapi-extension/issues/683 for more information.
